### PR TITLE
Remove nginx namespace limitation

### PR DESCRIPTION
**Pull request by pipo02mix**
**IMPORTANT: The changelog for the next release is generated automatically, based on pull requests merged to `master`. In order to properly categorize the changes made, you should allocate a proper label to your pull request:**

- Introducing a new feature

  - `kind/feature`

- Changing existing functionality

  - `kind/change`
  - `kind/ux-enhancement`

- Soon-to-be removed features

  - `kind/deprecation`

- Removing features

  - `kind/removal`

- Fixing bugs

  - `kind/bug`

- Fixing vulnerabilities
  - `kind/security`

**Excluded from changelog**

- `dependencies` (Updating dependencies)
- `kind/dev-change` (Changing development commands, file structure, etc.)
- `kind/refactor` (Improving the implementation of a specific part of the codebase, without changing its functionality)

### CODE REVIEW

File: __tests__/AppCatalog.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ describe('Apps and App Catalog', () => {
       });
 
       // Check if namespace input is disabled
-      expect(getByLabelText(/namespace:/i).readOnly).toBeTruthy();
+      expect(getByLabelText(/namespace:/i).readOnly).toBeFalsy();
```
Comment (pipo02mix) [inline __tests__/AppCatalog.js:290]: Now it should be editable

File: src/components/AppCatalog/AppDetail/InstallAppForm.js [hunk 2]
```diff
@@ -63,23 +63,22 @@ const InstallAppForm = ({
...
+    const hasFixedNamespace = false;
+    let appNamespace = '';
 
     // Some apps have special rules about what namespace they are allowed to be in.
     if (appName === 'nginx-ingress-controller-app') {
-      hasFixedNamespace = true;
-      fixedNamespace = 'kube-system';
-      updateNamespace(fixedNamespace);
+      appNamespace = 'kube-system';
+      updateNamespace(appNamespace);
```
Comment (pipo02mix) [inline src/components/AppCatalog/AppDetail/InstallAppForm.js:76]: We change from forcing to be `kube-system` to making it default value

**Review by oponder — APPROVED** [review]
LGTM!

**Comment by pipo02mix** [comment]
Closed in favour of https://github.com/giantswarm/happa/pull/1989#pullrequestreview-538371181

**Pull request merged**